src/desktop/components/inquiry_questionnaire/views/account.coffee

```diff
@@ -57,13 +57,31 @@ module.exports = class Account extends StepView
       @submit(form, recaptcha_token)
     )
 
+  handle2FAError: (form, response) ->
+    errorMessage = response.responseJSON.error
+    if errorMessage is "missing two-factor authentication code"
+      # reset form state
+      form.state.call(form, null)
+      form.reenable.call(form)
+      # reveal OTP field
+      form.$form.find('.iq-otp-field').removeClass('is-hidden')
+      wasHandled = true
```

Comment:
This grows the number of places where we act on the specific text of an error message. 

When we see the telltale error, we call methods on the form object to reset it to a submittable state, and reveal the OTP field